File: KthLargestElement.py
# Kth Largest Element
# - Given an integer array `nums` and an integer `k`, return the kth largest element.
# - Constraints:
#   - 1 <= k <= nums.length <= 10^4
#   - -10^4 <= nums[i] <= 10^4

from typing import List

class Solution:
    def findKthLargest(self, nums: List[int], k: int) -> int:
        nums.sort()
        return nums[-k]

File: KthLargestElement.cs
// Kth Largest Element
// - Given an integer array `nums` and an integer `k`, return the kth largest element.
// - Constraints:
//   - 1 <= k <= nums.Length <= 10^4
//   - -10^4 <= nums[i] <= 10^4

using System;

public class Solution
{
    public int FindKthLargest(int[] nums, int k)
    {
        Array.Sort(nums);

        return nums[nums.Length - k];
    }
}
